### DISCUSSION TITLE
第1节课笔记(16班)

### DISCUSSION THREAD

**Discussion by huzhanyu** [InternLM/Tutorial#34] [笔记]
请在下方提交笔记链接

**Comment by stydwn** [comment]
https://spurious-gondola-b7b.notion.site/Day01-e902ddcbf97748ce84b489b3575db429

**Comment by Wanghlearn** [comment]
https://github.com/Wanghlearn/SHPY/blob/main/notes-No1

**Comment by Aitejiu** [comment]
https://www.cnblogs.com/aitejiu/p/17944181

**Comment by Dhaizei** [comment]
https://blog.csdn.net/lishijie258/article/details/135373333?fromshare=blogdetail

**Comment by 2404589803** [comment]
https://2404589803.github.io/My-datawhale-learning/#/shusheng_puyu_learnning53/ch01

**Comment by Aorg** [comment]
#bilibili专栏# undefined - 哔哩哔哩专栏 https://www.bilibili.com/read/cv29000919/?spm_id_from=333.999.0.0

**Comment by future-2002** [comment]
https://blog.csdn.net/Y2483490891/article/details/135373571?spm=1001.2014.3001.5501

**Comment by Eadral** [comment]
https://www.yuque.com/eadral/ruaugi/gc2dxxl35bb1vvk5?singleDoc

**Comment by springkin** [comment]
https://airy-planarian-ede.notion.site/2fd91d4f19da42d5b7804e53fe663e66?pvs=4

**Comment by chrisx599** [comment]
https://carpal-mustang-f5e.notion.site/75733346dc2240f196a956b85ceb91ed

**Comment by xuanxiaochenmi** [comment]
https://github.com/xuanxiaochenmi/InternLLM_notes/blob/master/lesson_1/lesson_1.md

**Comment by TianCai19** [comment]
[第一天的笔记：书生大模型](https://codetwice.notion.site/day1-785d9e397ac441f39544ee70fd7e47a7?pvs=4)

**Comment by zhaoyiz0606** [comment]
https://blog.csdn.net/qq_45699866/article/details/135382029

**Comment by hhhanqian** [comment]
13 【书生·浦语大模型笔记（一） - 嗨嗨 | 小红书 - 你的生活指南】 😆 JbTvyqzIoS81ZiI 😆 http://xhslink.com/r56yFy

**Comment by Durian-1111** [comment]
Day01-LearningNote 
Link: https://xby48gs45u.feishu.cn/docx/SHCNdQ8lzoJlmqxEO7oceO7Anbh?from=from_copylink

**Comment by Star-cre** [comment]
https://www.bilibili.com/video/BV1Rc411b7ns/?vd_source=2fb638751797274bd22bea982387a179
b站公开笔记-戈壁的老王

**Comment by DavidAlpha007** [comment]
https://github.com/DavidAlpha007/DavidAlpha007/blob/63d1654ea7574dbdea47ea879eea5beea0191ceb/Notes%20of%20InternLM

**Comment by huzhanyu** [comment]
https://blog.csdn.net/qq_52711752/article/details/135392861?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135392861%22%2C%22source%22%3A%22qq_52711752%22%7D

**Comment by morcake** [comment]
https://blog.51cto.com/morcake/9102897

**Comment by EnableAsync** [comment]
https://enableasync.github.io/internlm/internlm-01

**Comment by xiaogege457** [comment]
https://www.yuque.com/mabingqi/gxtzsq/pn3h4yp1sqgta3w3?singleDoc# 《书生·浦语大模型训练营》

**Comment by Yang-Changhui** [comment]
https://blog.csdn.net/sdkhgsojglj/article/details/135395709

**Comment by yechenzhi** [comment]
https://docs.qq.com/doc/DZHFXd0xyTHhRZ1ZB

**Comment by INTERN-JLNG** [comment]
https://github.com/INTERN-JLNG/InternLLM/blob/main/Lec1_note.md

**Comment by zszbyzsz** [comment]
https://juejin.cn/post/7320245973107834880

**Comment by andrepan** [comment]
https://www.yuque.com/andrepan/eg7adh/gu0knk3dd89nbycu?singleDoc# 《InternLM》

**Comment by etheral12138** [comment]
https://y1yc0dwhb0o.feishu.cn/docx/Kqr9dMvlPoxdbJxuWkvc4Bu6ngh?from=from_copylink

**Comment by streamflowmaster** [comment]
https://www.yuque.com/shushizhenliu/ghur8i/wo0vy4yghleegscn
